app/bundles/EmailBundle/Views/Send/form.html.php

```diff
@@ -43,7 +43,10 @@
                             </div>
                             <?php echo $view['form']->errors($form['batchlimit']); ?>
                             <div class="text-center">
-                                <span class="label label-primary mt-lg"><?php echo $view['translator']->transChoice('mautic.email.send.pending', $pending, ['%pending%' => $pending]); ?></span>
+                                <span class="label label-primary mt-lg"><?php echo $view['translator']->trans(
+                                        'mautic.email.send.pending',
+                                        ['%pending%' => $pending]
```

Comment:
```suggestion
                                        ['%count%' => $pending]
```
plus update the ini file